cmd/launcher/main.go

```diff
@@ -348,6 +348,9 @@ func main() {
 		level.Info(logger).Log("msg", "beginnning shutdown")
 		return nil
 	}, func(err error) {
+		if err != nil {
+			level.Info(logger).Log("err", err)
```

Comment:
maybe include a `msg` here with where the error occurred? or just include `err` in the next log (and it can just be nil when it's nil)?

Reply:
I think if it's nil it panics, but good idea about the msg

Reply:
Sorry, I was wrong! I'll do that.